baselines/fedmlb/README.md

```diff
@@ -0,0 +1,627 @@
+    <td>49.15</td>
+  </tr>
+  <tr>
+    <td>FedAvg+KD</td>
+    <td>42.99</td>
+    <td>46.03</td>
+    <td>49.17</td>
+    <td>50.54</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td> 
+    <td>47.39</td>
+    <td>51.11</td>
+    <td>54.58</td>
+    <td>57.33</td>
+  </tr>
+</table>
+
+![cifar100, 100 clients, 5% participation rate, alpha = 0.3](_static/accuracy_comparison_cifar100_dirichlet_0.3_clients100.png)
+
+#### Tiny-ImageNet, Dir(0.3), 100 clients, 5% participation.
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>33.94</td>
+    <td>33.39</td>
+    <td>35.42</td>
+    <td>35.78</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>37.20</td>
+    <td>35.42</td>
+    <td>40.16</td>
+    <td>39.14</td>
+  </tr>
+</table>
+
+![tiny imagenet, 100 clients, 5% participation rate, alpha = 0.3](_static/accuracy_comparison_tiny-imagenet_dirichlet_0.3_clients100.png)
+
+
+### Table 1b and Table 2
+
+The results of Table 1a in the paper (for FedAvg and FedMLB) refers
+to CIFAR-100 (Figure 3) and Tiny-ImageNet (Figure 7b) with 
+the setting (2.) Large-scale experiments with Dir(0.3), 500 clients, 
+2% participation rate.
+
+To reproduce the results run the following:
+```bash
+python -m fedmlb.main --multirun dataset_config.dataset="cifar100","tiny-imagenet" algorithm="FedMLB","FedAvg","FedAvg+KD" total_clients=500 clients_per_round=10
+```
+#### CIFAR-100, Dir(0.3), 500 clients, 2% participation.
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>29.87</td>
+    <td>32.54</td>
+    <td>37.48</td>
+    <td>38.75</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>32.03</td>
+    <td>36.63</td>
+    <td>42.61</td>
+    <td>46.65</td>
+  </tr>
+</table>
+
+![cifar100, 500 clients, 2% participation rate, alpha = 0.3](_static/accuracy_comparison_cifar100_dirichlet_0.3_clients500.png)
+
+#### Tiny-ImageNet, Dir(0.3), 500 clients, 2% participation.
+
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>23.63</td>
+    <td>22.62</td>
+    <td>29.48</td>
+    <td>28.37</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>28.39</td>
+    <td>26.84</td>
+    <td>33.67</td>
+    <td>33.13</td>
+  </tr>
+</table>
+
+![tiny imagenet, 500 clients, 2% participation rate, alpha = 0.3](_static/accuracy_comparison_tiny-imagenet_dirichlet_0.3_clients500.png)
```

Comment:
```suggestion
#### CIFAR-100 and Tiny-ImageNet, Dir(0.3), 500 clients, 2% participation.

<table>
  <tr>
    <td></td>
    <td style="text-align: center", colspan="4">CIFAR-100</td>
    <td style="text-align: center", colspan="4">Tiny-ImageNet</td>
  </tr>
  <tr>
    <td></td>
    <td colspan="2">Accuracy @500R</td>
    <td colspan="2">Accuracy @1000R</td>
    <td colspan="2">Accuracy @500R</td>
    <td colspan="2">Accuracy @1000R</td>
  </tr>
  <tr>
    <td>Method</td>
    <td>Paper</td>
    <td>This repo</td>
    <td>Paper</td>
    <td>This repo</td>
    <td>Paper</td>
    <td>This repo</td>
    <td>Paper</td>
    <td>This repo</td>
  </tr>
  <tr>
    <td>FedAvg</td>
    <td>29.87</td>
    <td>32.54</td>
    <td>37.48</td>
    <td>38.75</td>
    <td>23.63</td>
    <td>22.62</td>
    <td>29.48</td>
    <td>28.37</td>
  </tr>
  <tr>
    <td>FedMLB</td>
    <td>32.03</td>
    <td>36.63</td>
    <td>42.61</td>
    <td>46.65</td>
    <td>28.39</td>
    <td>26.84</td>
    <td>33.67</td>
    <td>33.13</td>
  </tr>
</table>

<img src="_static/accuracy_comparison_cifar100_dirichlet_0.3_clients500.png" width="400"/> <img src="_static/accuracy_comparison_tiny-imagenet_dirichlet_0.3_clients500.png" width="400"/>

```

Reply:
It's great to have so many results to show in the `README.md`! but it also makes it a bit long. What i propose is to fuse tables together (same experiment for CIFAR-100, then TinyImageNet) and then show the plots side by side. What do you think?

Reply:
If you are happy with this, could you do the same for the `Table 1a and Table2` pair of results above?